packages/syft/src/syft/core/tensor/smpc/mpc_tensor.py

```diff
@@ -352,6 +353,78 @@ def __getattribute__(self, attr_name: str) -> Any:
 
         return object.__getattribute__(self, attr_name)
 
+    @staticmethod
+    def reshare(self: MPCTensor, parties: Iterable[Any]) -> MPCTensor:
```

Comment:
OUGHTTO: `staticmethod` should not have a `self` attribute attached to them - could you name it with something else?